### ISSUE TITLE
Clipboard seems to be broken (on Sway?)

### ISSUE THREAD

**Issue by etircopyh** [flameshot-org/flameshot#2626]
### Flameshot Version

v11.0.0

### Installation Type

Linux, MacOS, or Windows Package manager (apt, pacman, eopkg, choco, brew, ...)

### Operating System type and version

Arch Linux

### Description

Whenever I want to paste a screenshot in my Sway it only pastes text content. Yes, I have it built with -DUSE_WAYLAND_CLIPBOARD=1 and KGuiAddons installed.

### Steps to reproduce

1. run `flameshot gui`
2. select region
3. copy to clipboard
4. paste somewhere
5. get text or nothing (depending if you have text in your clipboard) / and wl-paste -t image/png also says that there's nothing

### System Information

1. Linux archdesk 5.17.9-arch1-1 #1 SMP PREEMPT Wed, 18 May 2022 17:30:11 +0000 x86_64 GNU/Linux
2. Laptop display 1366x768
3. Sway Wayland

**Comment by AndrewVos** [comment]
This is also happening for me - I'm not on Sway.

**Comment by AndrewVos** [comment]
@etircopyh Ok so this is weird that we both had this problem at the same time. My problem was actually that `notify-send` blocks up when trying to send notifications. This was because I wasn't launching `dunst` on startup.

Weirdly I was never launching `dunst` so I think the behaviour changed. It used to launch somehow whenever `notify-send` happened.
If you're using `dunst` then just launch it in a terminal and try the clipboard again.

**Comment by AndrewVos** [comment]
Oh wait, disregard that, clipboard is still not working in `flameshot`.

**Comment by prideandjoy5** [comment]
I am (and have been) having the same issue on Arch with KDE.  
I had "flameshot gui" as the hot key target and flameshot stopped working entirely a wile back.  I changed the hotkey to just flameshot and it starts, but I have to manually click on take screenshot.
While I can take a screenshot - the copy to clipboard action seemingly does nothing but closes out the capture screen.

**Comment by mmahmoudian** [comment]
@prideandjoy5 I don't think your issue has anything to do with what's reported here. As the matter of fact it cannot be even further apart. The op uses Wayland, you use X11, they use Sway WM, you use KDE DE. Please search in the issues and if you didn't find anything related, open a new bug report. Thank you.

**Comment by etircopyh** [comment]
> Can you test to see if you can get the correct results by piping the `--raw` into `wl-copy`?

Yes, this way it works. But `--clipboard` flag doesn't work at all and I'd like it to use it along with `--path` so I could save my screenshots locally as well as having them at the fingertips to use.

**Comment by utamadonny** [comment]
i have same problem, im using ubuntu 20.04 and set print hotkey as `flameshot gui`, the clipboard didnt work as this issues described. But, i notice i should open flameshot app so this icon pop up and the clipboard is work as expected. 
![image](https://user-images.githubusercontent.com/47313653/171331055-d339f572-1bfe-4a06-9c86-e21e74933d27.png)

im using flameshot snap version.

**Comment by mmahmoudian** [comment]
@utamadonny this is because you should run dbus beforehand. This is not only flameshot thing:

https://github.com/swaywm/sway/issues/4481

Also we have explained this here:

https://flameshot.org/docs/guide/troubleshooting/

**Comment by name-snrl** [comment]
I've the same problem, NixOS unstable, sway v1.7, Flameshot v11.0.0 with Qt 5.15.3

**Comment by borgmanJeremy** [comment]
This issue needs community involvement. I don't really support Sway or Nix

**Comment by etircopyh** [comment]
@borgmanJeremy, @AndrewVos reported problems with clipboard although not using Sway. But he didn't really clarify what's he using, X11 or Wayland 🤔

**Comment by AndrewVos** [comment]
Wayland

**Comment by zim0369** [comment]
Similar problem on "Arch with xmonad on X11". I think there could be an easy way to do this with `xclip` or `xsel`.

**Comment by mmahmoudian** [comment]
@zim0369 This thread is about Wayland. For your issue you have to read [this comment](https://github.com/flameshot-org/flameshot/issues/2526#issuecomment-1102277455). Of course you can **also** use `xclip` or `xsel` if you want. It is as simple as `flameshot gui --raw | xclip -sel clip`.

I'll mark your comment and this comment of mine as off-topic.

**Comment by fernandogrd** [comment]
It was not working for me, until I first opening flameshot and then running flameshot gui. By running flameshot gui before running just flameshot it would not work.
